Cargo.lock

```diff
@@ -3261,11 +3274,23 @@ dependencies = [
 
 [[package]]
 name = "serde_yaml"
-version = "0.9.34+deprecated"
+version = "0.8.26"
```

Comment:
What's the reason for this change? It seems to downgrade serde_yaml?

(Not sure I understand why this should happen because of the change you did.)

Reply:
This is automatic because the serde_yaml change is breaking and cargo manages to break everything because of it...